test/common/http/path_utility_test.cc

```diff
@@ -26,8 +27,33 @@ class PathUtilityTest : public testing::Test {
   TestRequestHeaderMapImpl headers_;
 };
 
+// Already normalized path don't change using deprecated path canonicalizer.
+TEST_F(PathUtilityTest, DeprecatedAlreadyNormalPaths) {
```

Comment:
I think you need to use the `DEPRECATED_FEATURE_TEST` here as it will fail in one of the builds.

Reply:
This is added. I actually not sure if we should go with "reloadable" flag or "deprecated_features" semantically. cc. @htuch.

Reply:
I think it should be `reloadable`. `deprecated_features` is for code changes that modify existing behavior.